packages/typescript-estree/src/convert.ts

```diff
@@ -2314,8 +2314,15 @@ export class Converter {
         return result;
       }
 
-      case SyntaxKind.ParenthesizedExpression:
+      case SyntaxKind.ParenthesizedExpression: {
+        if (ts.getJSDocTags(node).length > 0) {
```

Comment:
Let's check that at least one of the tags is `JSDocTypeTag`.

Reply:
ok i changed condition to

```suggestion
        if (ts.getJSDocTypeTag(node)) {
```